pkg/tag/tag_test.go

```diff
@@ -10,15 +10,15 @@ func TestFind(t *testing.T) {
 	if err != nil {
```

Comment:
this test in general is pretty legacy from before the rewrite and upstream, but it may benefit from having these inner cases as t.Run named subtests. Not the greatest test, but still reasonable to have some sanity checks like this.

Reply:
Done. Added a few extra test cases.